### PR TITLE
G2 2023 v3 issue #13400 test descriptions

### PR DESCRIPTION
**Pull request by c21alest**
Related to #13400

### CODE REVIEW

File: backend-models/service-tests/sectionedservice-tests.txt [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+----------------------------------
+deleteTest() line 181:
+----------------------------------
+
+*********
+ Pre-req
+*********
+
+(checklogin) IS TRUE
+AND $ha || $studentTeache IS TRUE
```
Comment (a21oscgu) [inline backend-models/service-tests/sectionedservice-tests.txt:37]: Did you mean "studentTeacher"? Check below aswell.

File: backend-models/service-tests/sectionedservice-tests.txt [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+*****************
+ Compare results
+*****************
+
+If gradesystem has been updated: passed
+If gradesystem not updated (same as beforeUpdated): failed
+If $debug not empty: failed
+
+----------------------------------
+updateVersTest() line 381:
```
Comment (a21oscgu) [inline backend-models/service-tests/sectionedservice-tests.txt:306]: Wrong line? Should be line 390?

**Review by a21oscgu — CHANGES_REQUESTED** [review]
Need some changes. See comments.

File: backend-models/service-tests/sectionedservice-tests.txt [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+
+* opt = "GRP"
+* courseid = "1"
+* coursevers = "45656"
+
+/////////////////////////
+
+
+*****************
+ Compare results
```
[the remark this answers is not in the document]
Comment (c21alest) [reply backend-models/service-tests/sectionedservice-tests.txt:23]: And the picture above is from the "Retrieve information" which we are not suppose to generate tests for, according to the README service-test.

[the remark this answers is not in the document]
Comment (a21jeaha) [reply backend-models/service-tests/sectionedservice-tests.txt:23]: as of yesterday, we no longer do comparisons on values before and after a service has been executed trough SQL-querys. 
we now perform the service with valid inputs, and save the resulting output.

this output will later be used, to compare outputs from the monolith to the output from the same services as micro services.

**Comment by c21alest** [comment]
This is an old duplicate of PR, new: #13637

**Pull request closed without merging**